packages/terra-list/src/ListItem.jsx

```diff
@@ -22,57 +28,73 @@ const propTypes = {
    */
   isSelectable: PropTypes.bool,
   /**
-   * Whether or not the list item has a disclosure indicator presented.
+   * The associated metaData to be provided in the onSelect callback.
    */
-  hasChevron: PropTypes.bool,
+  metaData: PropTypes.object,
+  /**
+   * Function callback for when the appropriate click or key action is performed.
+   * Callback contains the javascript evnt and prop metadata, e.g. onSelect(event, metaData)
+   */
+  onSelect: PropTypes.func,
   /**
    * Function callback for the ref of the li.
    */
   refCallback: PropTypes.func,
 };
+/* eslint-enable react/forbid-prop-types */
 
 const defaultProps = {
-  content: undefined,
+  children: [],
```

Comment:
Do we need a default value for children?

Reply:
I believe defaulting an array for children is our common pattern, but I don't know that it's required.